### ISSUE TITLE
HACS Support

### ISSUE THREAD

**Issue by danbuhler** [michaeldavie/flume-sensor#3]
Consider adding HACS support so it can be auto updated. It only needs basic changes to your file layout.

See https://custom-components.github.io/hacs/developer/integration/

**Comment by mcondren** [comment]
Yes please! This will make it much easier to test new builds for index out of range issue.

**Comment by DMQ8** [comment]
+1 for HACS support, Thanks.
